save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/AgentConfiguration.kt

```diff
@@ -53,14 +54,28 @@ data class AgentConfiguration(
     } else {
         id
     }
+
+    companion object {
+        /**
+         * @return [AgentConfiguration] with required fields initialized from env
+         */
+        internal fun initializeFromEnv() = AgentConfiguration(
```

Comment:
why we support only these fields in env?

Reply:
These are required fields with no default value, others can possibly be set in `updateFromEnv` later